### DISCUSSION TITLE
AnonSaints Jeremy White

### DISCUSSION THREAD

**Discussion by AnonSaintsGroup** [community/community#33197] [Mobile]
Yes I'm just getting getting together with the beta program I'm very platforms and I was just wondering if you could get confirmation of this message from one and two maybe an app with some kind of tools that would be useful if you have anything in mind it'd be greatly appreciated and have a good day

**Comment by ossanna16** [comment]
Hi @AnonSaintsGroup ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the topic specifically. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
